packages/suite/src/components/wallet/Menu/index.tsx

```diff
@@ -174,6 +175,11 @@ const Menu = ({
                     })
                 }
                 disabled={!!addAccountDisabled}
+                tooltipMessage={
+                    !device.connected ? (
```

Comment:
not sure about this, there are multiple reasons why "add account" is disabled
i think this tooltip (and its Tranlsation) should be dynamically defined inside AddAccount button

Reply:
and maybe `useDeviceActionLocks` should be added too?
maybe whole logic of `addAccountDisabled` should be moved into AddAccount ?

Reply:
yup, I agree there are more reasons why the button could be disabled. Feel free to expand it in another PR, It seemed a bit out of scope for this PR. I just wanted to fix broken tooltip position and noticed there was some nonsense message in tooltip.
So this only fixes the position and adds correct message for the disconnected device case

Reply:
since you already adding some logic add it properly. 
instead of sending `tooltipMessage` into AddAccount component, send whole `device` and do the if (device.connected) condition and Transaltion delaration inside `<AddAccount>`